udata_hydra/analysis/csv.py

```diff
@@ -115,6 +115,8 @@ class CustomDialect(stdcsv.unix_dialect):
 
 def smart_cast(_type, value, failsafe=False) -> Any:
     try:
+        if value is None or value == "":
```

Comment:
Why not `if not value`? Is there a value that doesn't match this criterion?

Reply:
I was not sure of `value` possible types, but I didn't want to return None if the value was not empty but evaluated to False (False, [], etc.).

Reply:
Makes sense 👍